src/components/Map/Map.tsx

```diff
@@ -70,14 +70,33 @@ const Map: FC<PropsWithChildren<MapProps>> = ({
     tap: false,
     scrollWheelZoom: false,
     center,
+    keyboard: false,
     ...mapOptions,
   }
 
   const dispatch = useDispatch()
   const { mapActive } = useSelector(makeSelectIncidentContainer)
 
   useEffect(() => {
+    /**
+     *  Set the link .leaflet-control-attribution a tabindex property -1, after
+     *  Leaflet map is fully loaded. This happens synchronous so surrounding with a timeout
+     *  0 will set the tabindex after the a element is loaded.
+     */
+    const timeout = setTimeout(() => {
+      // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+      // @ts-ignore
+      const leafletControlAttribution: HTMLLinkElement = document.querySelector(
+        '.leaflet-control-attribution a'
+      )
+
+      if (leafletControlAttribution) {
+        leafletControlAttribution.tabIndex = -1
+      }
+    }, 0)
+
```

Comment:
nogal hacky, maar blij dat het gelukt is

Reply:
haha ja echt heh